src/qibo_cloud_backends/braket_client.py

```diff
@@ -0,0 +1,133 @@
+import networkx as nx
+from braket.devices import LocalSimulator
+from qibo import Circuit as QiboCircuit
+from qibo.backends import NumpyBackend
+from qibo.config import raise_error
+from qibo.result import MeasurementOutcomes
+
+# Import Qiskit packages for transpiler
+from qiskit import QuantumCircuit, qasm2, transpile
+
+from qibo_cloud_backends.braket_translation import to_braket
+
+
+class BraketClientBackend(NumpyBackend):
+    def __init__(self, device=None, verbatim_circuit=False):
+        """Backend for the remote execution of AWS circuits on the AWS backends.
+
+        Args:
+            device (str): The ARN of the Braket device. Defaults to Braket's statevector LocalSimulator, LocalSimulator("default").
+                          Other devices are Braket's density matrix simulator, LocalSimulator("braket_dm"), or any other
+                          QPUs.
+            verbatim_circuit (bool): If `True`, to_braket will wrap the Braket circuit in a verbatim box to run it on the QPU
+                                     without any transpilation. Defaults to `False`.
+        """
+        super().__init__()
+
+        self.verbatim_circuit = verbatim_circuit
+
+        self.device = device if device else LocalSimulator()
+        self.name = "aws"
+
+    @staticmethod
+    def custom_connectivity(coupling_map):
+        """Converts a coupling map given in list form to a networkx graph. Returns networkx graph.
+
+        Args:
+            coupling_map (list): E.g. [[0, 1], [0, 7], [1, 2], [2, 3], [4, 3], [4, 5], [6, 5], [7, 6]]
+        Returns:
+            graph (networkx graph): graph
+        """
+
+        graph = nx.Graph()
+        for connection in coupling_map:
+            q1, q2 = connection
+            graph.add_edge(q1, q2)
+        return graph
+
+    def transpile_qibo_to_qibo_with_qiskit(
```

Comment:
Again, this should not be part of the backend in my view. Transpilation should be separated, and, probably delegated to the user. Furthermore, we should use, as much as possible, the transpiler we have in qibo.

Reply:
Yes, agree. We had difficulty using Qibo's transpiler at the start. So we used Qiskit's transpiler as it was fairly straightforward. I will remove it as it does not belong to the backend. I agree that transpilation is the end-user's responsibility.